script/playbackstrategy/msestrategy.js

```diff
@@ -120,6 +141,55 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
         }
       }
 
+      function createPlaybackProperties () {
```

Comment:
This would read a lot clearer if you returned an object literal rather than created an empty object then adding the keys